### PR TITLE
Fix to make ONNXIFI flow work

### PR DESCRIPTION
**Pull request by yinghai**
Small step to have Relu test work. 

Test Plan:
- Patch https://github.com/onnx/onnx-tensorrt/pull/14 and build `libtrt_onnxify.so`
- `ln -s /home/yinghai/onnx-tensorrt/build/libtrt_onnxify.so ~/lib/libonnxifi.so`
- `export LD_LIBRARY_PATH=/home/yinghai/lib:$LD_LIBRARY_PATH`
- Patch this PR and build Pytorch
- `pytest -s test_onnxifi.py`

### CODE REVIEW

File: caffe2/python/onnx/test_onnxifi.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        X = np.random.randn(batch_size, 1, 3, 2).astype(np.float32)
+        graph_def = make_graph(
+            [make_node("Relu", ["X"], ["Y"])],
+            name="test",
+            inputs=[make_tensor_value_info("X", onnx.TensorProto.FLOAT,
+                [batch_size, 1, 3, 2])],
+            outputs=[make_tensor_value_info("Y", onnx.TensorProto.FLOAT,
+                [batch_size, 1, 3, 2])])
+        model_def = make_model(graph_def, producer_name='relu-test')
+        op = core.CreateOperator(
```
Comment (bddppq) [inline caffe2/python/onnx/test_onnxifi.py]: Do you need to set engine/device_type to make it to run with trt?

Comment (Maratyszcza) [reply caffe2/python/onnx/test_onnxifi.py]: @bddppq: right now OnnxifiOp just uses the first available backend

Comment (bddppq) [reply caffe2/python/onnx/test_onnxifi.py]: @Maratyszcza so caffe2 unconditionally delegate to onnxifi backends?

Comment (Maratyszcza) [reply caffe2/python/onnx/test_onnxifi.py]: OnnxifiOp unconditionally delegate to ONNXIFI backend - thats what it is supposed to do

Comment (bddppq) [reply caffe2/python/onnx/test_onnxifi.py]: Oh nvm I thought this is a relu op

**Pull request closed without merging**